### PR TITLE
Added an Example app

### LINKED ISSUE #43: Add example java project

### PR DESCRIPTION
**Pull request by btkelly**
This just adds the empty example app, another issue has been created to fill it with violations. Resolves #43

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -2,79 +2,13 @@ buildscript {
     repositories {
         jcenter()
     }
-}
+    dependencies {
```
Comment (stkent) [inline build.gradle]: Kill this empty closure?

File: example/build.gradle [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath fileTree("../plugin/build/libs")
+    }
+}
+
+plugins {
+    id "com.github.hierynomus.license" version "0.12.1"
```
Comment (stkent) [inline example/build.gradle:11]: You can apply this in the root build.gradle file; see BugShaker.

**Comment by stkent** [comment]
Looks good apart from that comment. You could make a fake gradle task to wrap the shell script for building the example, if you wanted?

File: example/build.gradle (hunk 2 above)
Comment (btkelly) [inline example/build.gradle:11]: So in this case you cannot as the example application is not linked with the root build.gradle file. This was on purpose as it allows the plugin to be built without evaluating the existence of the plugin for the example project.

Comment (stkent) [inline example/build.gradle:11]: Ahh, interesting, makes sense!

**Comment by stkent** [comment]
LGTM on success!

**Pull request merged**